Sources/_TestingInternals/include/Stubs.h

```diff
@@ -180,6 +180,17 @@ static int swt_setfdflags(int fd, int flags) {
 }
 #endif
 
+#if !SWT_NO_INTEROP
+
+typedef void (*FallbackEventHandler)(const char *recordJSONSchemaVersionNumber,
```

Comment:
This PR's still a draft, but if you stick with this approach, please plan to add some documentation for these (even if it's a cursory explanation of how they shadow functions elsewhere).

Reply:
```suggestion
typedef void (* FallbackEventHandler)(const char *recordJSONSchemaVersionNumber,
```

The smallest possible nitpick. :)